online-bookstore/src/main/java/group1/itschool/onlinebookstore/models/dto/BookDTO.java

```diff
@@ -5,6 +5,20 @@
 @Data
 public class BookDTO {
 
-    private String name;
-    private String category;
+    private String title;
+    private String author;
+    private String publisher;
+    private int publicationDate;
+    private int codeISBN;
```

Comment:
Please add spring validation to this field. An isbn number has between min 10 and max 13 numbers.